java/meterpreter/stdapi/src/main/java/com/metasploit/meterpreter/stdapi/stdapi_sys_eventlog.java

```diff
@@ -0,0 +1,34 @@
+package com.metasploit.meterpreter.stdapi;
+
+import com.sun.jna.Library;
+import com.sun.jna.Native;
+import com.sun.jna.Pointer;
+import com.sun.jna.Platform;
+import com.sun.jna.ptr.IntByReference;
+
+import com.sun.jna.win32.W32APIOptions;
+
+public class stdapi_sys_eventlog {
+    interface AdvAPILibrary extends Library {
+        AdvAPILibrary INSTANCE = Native.load(
+                (Platform.isWindows() ? "Advapi32" : "THIS SHOULD NEVER BE CALLED."), AdvAPILibrary.class, W32APIOptions.DEFAULT_OPTIONS);
```

Comment:
Elsewhere in java meterpreter `File.pathSeparatorChar == ';'` is used as an IsWindows() function. e.g:
https://github.com/rapid7/metasploit-payloads/blob/master/java/meterpreter/stdapi/src/main/java/com/metasploit/meterpreter/stdapi/stdapi_sys_process_get_processes.java#L26
It might be worth pulling it's usage out into Utils.java or somewhere global and re-using it everywhere.
I can send you a pr with this change if needed
Nice work btw!

Reply:
https://github.com/sjanusz-r7/metasploit-payloads/pull/1